R/get_data.R

```diff
@@ -10,9 +10,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and limitations under the License.
 
-#' Download and read a dataset from a B.C. Data Catalogue resource
+#' Download and read a dataset from a B.C. Data Catalogue record
```

Comment:
I think this should still be a resource. `bcdc_get_data` ultimately returns a resource

Reply:
For this sentence though, the data set is the resource from  catalogue record? The most consistent with the rest of the documentation might be "Download and read a resource from a B.C. Data Catalogue record"?

Reply:
There it is. That is the clarity we are looking for.